evenodd.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main()
+{
+	int UserIn;
+	printf("Type a number: ");
+	scanf("%d", &UserIn);
+	if(UserIn%2 ==0)
+	{
+	printf("The number you typed is even\n");
+	}
+
+	else
+	{
+		printf("The number you typed is odd\n");
+	}
```

Comment:
Good code, but you don't follow basic guidelines of commenting or having a return 0; These are important!!